build/commands/lib/config.js

```diff
@@ -778,6 +778,9 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
     if (this.use_goma && this.gomaServerHost) {
       env.CC_WRAPPER = path.join(this.depotToolsDir, '.cipd_bin', 'gomacc')
       env.GOMA_SERVER_HOST = this.gomaServerHost
+      if (this.gomaJValue > 100) {
+        this.gomaJValue = 100
+      }
```

Comment:
Instead of changing this value here, could we just limit it at the same place it's calculated?

Line 131:
```
this.gomaJValue = Math.min(100, (os.cpus().length + 1) * 3)
```

Reply:
implemented